flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -1010,9 +1013,11 @@ private void clearStateInternal() {
      * manager has failed.
      *
      * @param jobId identifying the job whose leader shall be disconnected.
+     * @param resetRequirements whether to reset the resource requirements for the job to 0
```

Comment:
Maybe `clear` is clearer.

```suggestion
     * @param clearRequirements whether to reset the resource requirements for the job to 0
```